src/test/java/com/zufar/onlinestore/cart/stub/CartDtoTestStub.java

```diff
@@ -0,0 +1,224 @@
+        ProductInfoDto productInfoDto = new ProductInfoDto();
+        productInfoDto.setId(UUID.fromString("a834c24e-886d-470f-bf19-7454a60f0639"));
+        productInfoDto.setName("Test name");
+        productInfoDto.setDescription("Test description");
+        productInfoDto.setPrice(BigDecimal.valueOf(1.1));
+        productInfoDto.setQuantity(3);
+        productInfoDto.setActive(true);
+        shoppingSessionItemDto.setProductInfo(productInfoDto);
+
+        return shoppingSessionItemDto;
+    }
+
+    public static ShoppingSession createShoppingSession() {
+        UUID userId = UUID.fromString("2eebb17c-5a55-43dd-add7-c15d49521f14");
+        ShoppingSession shoppingSession = new ShoppingSession();
+        Set<ShoppingSessionItem> items = new HashSet<>();
+
+        UUID firstProductId = UUID.fromString("a834c24e-886d-470f-bf19-7454a60f0639");
+        ProductInfo firstProductInfo = new ProductInfo(
+                firstProductId, "First test name", "First test description", BigDecimal.valueOf(1.1), 1, true);
+        UUID secondProductId = UUID.fromString("2ade78e3-aa45-4b6b-adf4-86f8302ced7d");
+        ProductInfo secondProductInfo = new ProductInfo(
+                secondProductId, "Second test name", "Second test description", BigDecimal.valueOf(2.2), 2, true);
+        UUID thirdProductId = UUID.fromString("b58ac6f1-7ee1-4888-9055-3bebb6aa3631");
+        ProductInfo thridProductInfo = new ProductInfo(
+                thirdProductId, "Third test name", "Third test description", BigDecimal.valueOf(3.3), 3, true);
+
+        UUID firstItemId = UUID.fromString("9b588163-b781-46bf-8714-bd0145337ddc");
+        ShoppingSessionItem firstItem = new ShoppingSessionItem(
+                firstItemId, 1, shoppingSession, firstProductInfo, 1);
+        UUID secondItemId = UUID.fromString("e5cadeb1-089c-430f-85d1-e18438167241");
+        ShoppingSessionItem secondItem = new ShoppingSessionItem(
+                secondItemId, 1, shoppingSession, secondProductInfo, 2);
+        UUID thirdItemId = UUID.fromString("b00ed4dc-62d1-449c-b559-65d9c2cad906");
+        ShoppingSessionItem thirdItem = new ShoppingSessionItem(
+                thirdItemId, 1, shoppingSession, thridProductInfo, 3);
+
+        items.add(firstItem);
+        items.add(secondItem);
+        items.add(thirdItem);
+
+        shoppingSession.setUserId(userId);
+        shoppingSession.setItems(items);
+        shoppingSession.setItemsQuantity(3);
+        shoppingSession.setProductsQuantity(6);
+        shoppingSession.setCreatedAt(OffsetDateTime.now());
+        shoppingSession.setClosedAt(OffsetDateTime.now().plusHours(2));
+
+        return shoppingSession;
+    }
+
+    public static ShoppingSessionDto createShoppingSessionDto () {
+        UUID userId = UUID.fromString("ebd4d43f-3152-4af5-86dd-526a002cbbc3");
+        ShoppingSessionDto shoppingSessionDto = new ShoppingSessionDto();
+        List<ShoppingSessionItemDto> items = new ArrayList<>();
```

Comment:
Create a separate method which returns `List<ShoppingSessionItemDto> items`
And then use this method in tests

Reply:
done